app/views/shared/_user_info.html.erb

```diff
@@ -12,6 +12,10 @@
     <span class="text-hamers-red-400"><%= current_user.stickers.count %></span> stickers geplakt,
     <span class="text-hamers-red-400"><%= current_user.blogitems.count %></span> blogs geschreven en
     bent bij <span class="text-hamers-red-400"><%= Signup.where(event_id: Event.where(attendance: true).pluck(:id), user: current_user).count %></span>
-    borrels geweest.
+    borrels geweest, met een inschrijfpercentage van <span class="text-hamers-red-400"><%= current_user.drink_ratio %>%</span> in het afgelopen jaar.
+
+    <% if current_user.drink_ratio < 0.8 %>
```

Comment:
Als mensen nog niet aanwezig zijn geweest op een borrel (aka 0.0%), is deze if ook true. 
Weet niet echt of we dat willen of niet. 
Daarnaast twijfel ik aan de komma of deze wel of niet klopt